.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
```

Comment:
Kindly Make sure there is absolutely no liners error before requesting review. A code with no linter error indicate that you have follow the required and best practice of writing code.